### PR TITLE
[WAF] Add missing Content-Type headers

### PR DESCRIPTION
**Pull request by pedrosousa**

### CODE REVIEW

File: content/waf/managed-rules/check-for-exposed-credentials/configure-api.md [hunk 1]
```diff
@@ -105,17 +106,18 @@ highlight: 15-18
...
+The example uses the `url_decode()` function because fields in the request body (available in `http.request.body.form`) are URL-encoded when the content type is `application/x-www-form-urlencoded`.
 
 After creating a custom ruleset, deploy it to a phase so that it executes. Refer to [Deploy a custom ruleset](/ruleset-engine/custom-rulesets/deploy-custom-ruleset/) for more information.
 
 ### Example B
 
-This example creates a new custom ruleset with a rule that checks for exposed credentials in JSON responses. The rule has a match if both the rule expression and the `exposed_credential_check` result are `true`. When there is a match, the rule will add an `Exposed-Credential-Check` HTTP header to the request with value `1`.
+This example `POST` request creates a new custom ruleset with a rule that checks for exposed credentials in JSON responses. The rule has a match if both the rule expression and the `exposed_credential_check` result are `true`. When there is a match, the rule will add an `Exposed-Credential-Check` HTTP header to the request with value `1`.
```
Comment (marciocloudflare) [inline content/waf/managed-rules/check-for-exposed-credentials/configure-api.md]:
```suggestion
This `POST` request example creates a new custom ruleset with a rule that checks for exposed credentials in JSON responses. The rule has a match if both the rule expression and the `exposed_credential_check` result are `true`. When there is a match, the rule will add an `Exposed-Credential-Check` HTTP header to the request with value `1`.
```

Comment (marciocloudflare) [reply content/waf/managed-rules/check-for-exposed-credentials/configure-api.md]: I think it's a bit easier to understand if we write it like this. Feel free to ignore, though

**Review by marciocloudflare — APPROVED** [review]

File: content/waf/managed-rules/check-for-exposed-credentials/configure-api.md [hunk 2]
```diff
@@ -43,11 +43,12 @@ To create and deploy a custom ruleset, follow the workflow described in [Work wi
 
 ### Example A
 
-This example creates a new custom ruleset with a rule that checks for exposed credentials. The rule has a match if both the rule expression and the `exposed_credential_check` result are `true`. When there is a match, the rule will log the request with exposed credentials in the Cloudflare logs.
+This example `POST` request creates a new custom ruleset with a rule that checks for exposed credentials. The rule has a match if both the rule expression and the `exposed_credential_check` result are `true`. When there is a match, the rule will log the request with exposed credentials in the Cloudflare logs.
```
Comment (pedrosousa) [inline content/waf/managed-rules/check-for-exposed-credentials/configure-api.md]:
```suggestion
This `POST` request example creates a new custom ruleset with a rule that checks for exposed credentials. The rule has a match if both the rule expression and the `exposed_credential_check` result are `true`. When there is a match, the rule will log the request with exposed credentials in the Cloudflare logs.
```

File: content/waf/managed-rules/payload-logging/configure-api.md [hunk 3]
```diff
@@ -38,7 +38,7 @@ You can generate a public key [in the command line](/waf/managed-rules/payload-l
 
 ### Example
 
-The following example updates rule `{rule_id_1}` that executes the Cloudflare Managed Ruleset for zone `{zone_id}`, configuring payload logging with the provided public key.
+The following example `PATCH` request updates rule `{rule_id_1}` that executes the Cloudflare Managed Ruleset for zone `{zone_id}`, configuring payload logging with the provided public key.
```
Comment (pedrosousa) [inline content/waf/managed-rules/payload-logging/configure-api.md]:
```suggestion
The following `PATCH` request example updates rule `{rule_id_1}` that executes the Cloudflare Managed Ruleset for zone `{zone_id}`, configuring payload logging with the provided public key.
```

File: content/waf/managed-rules/check-for-exposed-credentials/configure-api.md (hunk 1 above)
Comment (pedrosousa) [reply content/waf/managed-rules/check-for-exposed-credentials/configure-api.md]: I updated other occurrences in this PR. Thanks!

**Pull request merged**